### PR TITLE
SSBFFA: Fix Move Discrepencies

### PR DESCRIPTION
**Pull request by ghost**
in tandem with #283

### CODE REVIEW

File: spacialgaze-plugins/ssb.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ let ssbWrite = true; //if false, do not write to json
...
+let customDescs = ['+1 Atk, +1 SpA, +1 Spe', '80 power Special attack, traps opponent for 4-5 turns and damages, 50% chance of burn', '50 power special move, 100 accuracy, summons rain, 20% chance to flinch', 'Heal your whole team of status conditions and heal 25% of your HP.', 'More power the faster the user is than the target, rasies speed by 1 after use.', 'Hail + Blizzard', '200 Base Power, has a 50% chance to paralyze target, must recharge after use', 'Inflict toxic on foe, and lower foes attack by 1.', '140BP Physical move, 15% chance to flinch', 'Remove entry hazards and set the weather to clear.', 'Sets Light Screen and Reflect.', '100 power physical attack, 90 accuracy, 30% chance to raise speed and attack.', 'Special attack, 110 power, 100 accuracy, 30% chance to Flinch', '70BP, 30% flinch chance, Always crits', '175BP outrage, also lowers your atk by 2 after it ends.', '100BP knock off', '100BP Physical move, if the foe is a steel type they will be trapped.', '120BP Special move. 30% par chance, powder based move.'];
```
Comment (HoeenCoder) [inline spacialgaze-plugins/ssb.js:8]: Now that we can do `/dt movename, cssb`, we should modify the popup to display the list of c move names, with a button to run `/dt movename, cssb` and the button to set the move.

Comment (DeathlyPlays) [reply spacialgaze-plugins/ssb.js:8]: Ooooh good idea

Comment (ghost) [reply spacialgaze-plugins/ssb.js:8]: I agree

Comment (HoeenCoder) [reply spacialgaze-plugins/ssb.js:8]: Feel free to PR that

Comment (DeathlyPlays) [reply spacialgaze-plugins/ssb.js:8]: Yeah it'd be HTML'ed so a display edit would be needed

Comment (HoeenCoder) [reply spacialgaze-plugins/ssb.js:8]: Not hard to do tbh

**Pull request merged**